src/go/wsl-helper/cmd/kubeconfig.go

```diff
@@ -45,10 +45,14 @@ var kubeconfigCmd = &cobra.Command{
 		enable := kubeconfigViper.GetBool("enable")
 		show := kubeconfigViper.GetBool("show")
 
+		if configPath == "" {
+			return errors.New("Windows kubeconfig not supplied")
+		}
+
 		_, err := os.Stat(configPath)
 		if err != nil {
 			if errors.Is(err, os.ErrNotExist) {
-				return fmt.Errorf("Could not open Windows kubeconfig: %w", err)
+				return fmt.Errorf("could not open Windows kubeconfig: %w", err)
```

Comment:
I don't understand why we are testing for `os.ErrNotExist` and are only prefixing that error with "could not open Windows kubeconfig". Why is that prefix not applied to other error conditions?

Reply:
I agree this is weird, so I've updated it to always return a wrapped error.